mantidproject/mantid
# Update ParaView to v5.3.0

Description of work.

This updates Mantid to use [ParaView 5.3.0](https://blog.kitware.com/paraview-5-3-0-release-notes/). It also updates vtkMDHWSignal array to use the newer [data array layouts](https://blog.kitware.com/new-data-array-layouts-in-vtk-7-1/) in VTK. In order to see a performance improvement, the header needs to be passed to VTK at compile time.

**To test:**



1. View the updated [build instructions](https://www.mantidproject.org/Building_VATES). Note that `vtkMDHWSignalArray.h` is now in the paraview-build repository and must be available to ParaView during compilation.
2. Build Mantid with -DMAKE_VATES=ON and -DParaView_DIR=<your ParaView build directory>.
3. Try viewing a MDHistoWorkspace and a MDEventWorkspace in the VSI.

There is no GitHub issue associated with this pull request.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
